aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -200,6 +200,7 @@ module aptos_token::token {
         withdraw_events: EventHandle<WithdrawEvent>,
         burn_events: EventHandle<BurnTokenEvent>,
         mutate_token_property_events: EventHandle<MutateTokenPropertyMapEvent>,
+        mutate_token_uri_events: EventHandle<MutateTokenUriEvent>
```

Comment:
unfortunately we can no longer modify this without introducing a breaking change, which we're no longer doing.

also note that the change affects the collections portion of data... let's strip out the event for now, most likely we'll need to create a new resource for this. we're in the midst of a post freeze audit to find other, similar issues.

Reply:
Would be possible to have a getter & setter for the token uri? it would be really appreciated as it is needed for nft revealing

Reply:
Yes new public functions are backward-compatible so it's fine to add them. You just can't add the event to an existing struct unfortunately. For now, let's skip the event, we can add a new struct with some missing events soon.

Reply:
sorry for the confusion. to make the PR simpler and easier to land, let's not worry about the events. Please remove all events. I will add events later in a batch.